src/utils/hash-link-scroll.js

```diff
@@ -19,7 +19,7 @@ function reset(observer, asyncTimerId) {
 function scrollToElement(id) {
   const element = document.getElementById(id)
   if (element) {
-    element.scrollIntoView()
+    setTimeout(() => { element.scrollIntoView() }, 200)
```

Comment:
Set the delay by function parameter or outside the function to make the delay time more flexiable? Example useage:

```js
const id = 'element-id'
const delay = 200  // Maybe the duration time of other effect. Wait until it's end.
scrollToElement(id, delay)
```

or


```js
const id = 'element-id'
const delay = 200
setTimeout(() => { scrollToElement(id) }, delay)
```

Reply:
Updated.